doc/policy/mempool-replacements.md

```diff
@@ -0,0 +1,65 @@
+# Mempool Replacements
+
+## Current Replace-by-Fee Policy
+
+A transaction conflicts with an in-mempool transaction ("directly conflicting transaction") if they
+spend one or more of the same inputs. A transaction may conflict with multiple in-mempool
+transactions.
+
+A transaction ("replacement transaction") may replace its directly conflicting transactions and
+their in-mempool descendants (together, "original transactions") if, in addition to passing all
+other consensus and policy rules, each of the following conditions are met:
+
+1. The directly conflicting transactions must all signal replaceability explicitly. A transaction is
```

Comment:
The "must", "may", etc words in these conditions are redundant with the sentence above and can be removed

```
... all of the following conditions are met:

1. The directly conflicting transactions all signal...
2. The replacement transaction only includes ...
```

Reply:
Removed